src/ai-store/generative-ai-service.js

```diff
@@ -290,7 +290,7 @@ class BrowserGenerativeAiService extends GenerativeAiService {
 			}
 		}
 
-		const session = await window.ai.createTextSession( modelParams );
+		const session = await self.ai.assistant.create( modelParams );
```

Comment:
Same here:
```suggestion
		const session = await window.ai.assistant.create( modelParams );
```

FWIW I personally don't know what's the difference between using `self` and `window` for this kind of access - whether it's just a preference or has actual behavioral impact?